sklearn/metrics/_classification.py

```diff
@@ -740,10 +740,14 @@ def jaccard_score(
     -------
     score : float (if average is not None) or array of floats, shape =\
             [n_unique_labels]
+        The Jaccard score.
```

Comment:
```suggestion
        The Jaccard score. When `average` is not `None`, a single scaler is
        returned.
```

Reply:
Do you mean `scalar`? Or is it actually `scaler`?

Reply:
Oh right. I meant `scalar`. I edited the suggestion.